src/pages/signin/SAMLSignInPage/index.native.tsx

```diff
@@ -13,9 +13,10 @@ import * as Session from '@userActions/Session';
 import CONFIG from '@src/CONFIG';
 import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
-import type {SAMLSignInPageOnyxProps, SAMLSignInPageProps} from './types';
 
-function SAMLSignInPage({credentials, account}: SAMLSignInPageProps) {
+function SAMLSignInPage() {
+    const [account] = useOnyx(ONYXKEYS.ACCOUNT);
```

Comment:
Do we need a default for `isLoading` or a null coalesce below? Why did you do `[account = {}]` above and `[account]` here?